electron/settingsRenderer.js

```diff
@@ -263,6 +280,101 @@ var firstCounterAdjButtonClicked = true;
 document.addEventListener("DOMContentLoaded", function(event) {
   rivets.bind(document.getElementById('container'), settingsData)
 
+  let collectionWorker = buildWorker(e => {
+    var allCards;
+    var cardSets = {}
+    var playerCardCounts = {}
+    onmessage = event => {
+      if (event.data.allCards) {
```

Comment:
Why aren't we just calculating this once and storing it?

Reply:
That's pretty much what is happening here; the value is only sent to the webworker once. This is def a hack that's explained in the chunk of comments below.

Reply:
No I mean, why don't we go "Ok there are 64 Mythics in Ixalan, 224 Rares..." in development then store that info in a json object for reading in production. No need to have the webworker ever calculate it.

Reply:
:shrug: it's not a bad idea, but I do like the idea of letting the mtga module be the authoritative source. (And, in this case, even if the mtga module did calculate those numbers, we'd miss out on it in webworkers--that is, unless it's a static number hardcoded in, which I wouldn't be too excited about)

Reply:
I can see that argument. I can also see the argument, however, that it's much easier to maintain an object that gets updated once every 4 months than it is to maintain some code. Also, less processing generally better. Now I don't feel strongly on this. Just thought I'd offer this opinion on it.